datafusion/physical-expr/src/equivalence/ordering.rs

```diff
@@ -29,11 +29,11 @@ use crate::{LexOrdering, PhysicalExpr, PhysicalSortExpr};
 /// |a|b|c|d|
 /// |1|4|3|1|
 /// |2|3|3|2|
-/// |3|1|2|2|
-/// |3|2|1|3|
+/// |3|2|2|2|
+/// |3|1|1|3|
 /// ```
 ///
-/// Here, both `vec![a ASC, b ASC]` and `vec![c DESC, d ASC]` describe the table
```

Comment:
I don't see `b` is ASC. It is more likely the original intension is to make it `DESC`. So I made the change.

Reply:
I think the original example is not `b ASC` itself, but it is `a ASC, b ASC` (that is it is first sorted by `a ASC` and then for rows where `a` is the same then the values are sorted by `b ASC`)

Reply:
Oh, you're right. That's correct. I misread it. Thanks.